docs/README.md

```diff
@@ -1920,60 +1914,277 @@ mutation {
         }
+      })
+  }
+  return forward(operation)
+})
+
+const httpLinkWithAuth = authLink.concat(httpLink) 
+
+const subscriptionClient = new SubscriptionClient('<websocketURL>', {
+  reconnect: true,
+  // CHANGED: added a "connectionParam" property with a function that returns the `Authorizaiton` header containing our token
+  connectionParams: () => {
+    if (authToken) {
+      return {
+        Authorization: `Bearer ${authToken}` // <-- CHANGED: We use the "authToken" global variable
       }
-    `,
+    }
+    return {}
+  }
 })
+const wsLink = new WebSocketLink(subscriptionClient);
+
+const splitLink = split(isSubscriptionOperation, wsLink, httpLinkWithAuth);
+
+const client = new ApolloClient({
+  link: splitLink,
+  cache: new InMemoryCache()
+});
 ```
 
+Now, _when the user signs-in_ or _when the token is refreshed_, we need to do two things:
+1. Update the global variable `authToken` with the new token.
+2. Reconnect the socket used by the subscription client by doing `subscriptionClient.close(false)`
+
+You might be wondering why we need to do the second step. The reason is that, with operations sent through HTTP, the token goes along with every operation, in the headers. However, with operations sent through WebSockets, like subscriptions, the token is only sent when the socket connection is established. For this reason, **everytime we update the token we need to reconnect the `SubscriptionClient`** so that it sends again the token (the updated one in this case).
+
+#### The GraphQL over WebSocket protocol
+
+Sockets are channels for two-way communication that doesn't follow the request-response cycle, a characteristic feature of the HTTP protocol. One part can send many messages and the other part could receive all of them but only answer to some specific ones. What is more, messages could come in any order. For example, one part can send two messages and receive the response of the second message before the response of the first message.
+
+For these reasons, in order to have an effective non-trivial communication through sockets, a sub-protocol is needed. It would be in charge of making both parts understand each other, share authentication tokens, matching response to the corresponding requests, etc.
+
+The Booster WebSocket communication uses the "GraphQL over WebSocket" protocol as subprotocol. It is in charge of all the low level stuff needed to properly send subscription operations to read models and receive the corresponding data.
+
+You don't need to know anything about this to develop usin Booster, neither in the backend side nor in the frontend side (as all the Apollo GraphQL clients uses this protocol), but it is good to know it is there to guarantee a proper communication. In case you are really curious, you can read about the protocol [here](https://github.com/apollographql/subscriptions-transport-ws/blob/master/PROTOCOL.md) 
```

Comment:
This `The GraphQL over WebSocket protocol` section is all that I meant in my previous comment. It is providing context for newcomers to web sockets, and experienced readers can omit the first two paragraphs. Then, an explanation about why that is relevant for understanding Booster, and finally, then readers, regardless of their expertise, can follow our link for more details about it.

I really loved ❤️  these four paragraphs